quickwit/quickwit-doc-mapper/src/doc_mapping.rs

```diff
@@ -99,6 +100,13 @@ impl Default for Mode {
 #[derive(Clone, Debug, PartialEq, Serialize, Deserialize, utoipa::ToSchema)]
 #[serde(deny_unknown_fields)]
 pub struct DocMapping {
+    /// Doc mapping UID.
+    ///
+    /// Splits with the same doc mapping UID share the same schema and should use the same doc
+    /// mapper during indexing and querying.
```

Comment:
am i right understanding that splits actually do not contain a doc mapping uid yet?

Reply:
Yes